curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-001.md

```diff
@@ -1,15 +1,13 @@
 ---
-id: 6193d8081ec2531581ea7b98
+id: 61fd5a93fd62bb35968adeab
 title: Step 1
 challengeType: 0
 dashedName: step-1
 ---
 
 # --description--
 
-A balance sheet is used to track the state of an organization's finances. You'll be building a mock balance sheet for AcmeWidgetCorp to explore the use of various CSS selectors.
-
-Begin with your basic HTML boilerplate. Include the `DOCTYPE` declaration, an `html` element, the appropriate `meta` tags, a `head`, `body`, and `title` element, and `link` your stylesheet. Give the `title` element the text `AcmeWidgetCorp Balance Sheet`.
+Set up your HTML with the `DOCTYPE`, `html`, `head`, and `body` elements. Give your `head` element the appropriate `meta` elements for the `charset` and `viewport`, a `title` element with an appropriate title, and a `link` element for your stylesheet.
```

Comment:
This is the case, then, for most of our challenges. The alternative would be regex, but ew.

Reply:
> I just accept the tests are passing because the final output is correct

For the most part, I would agree - it's easier and more reliable to check the output rather than the code - particularly with HTML/CSS. When you get to JS, it seems like there might be more times where you want to check the syntax. I do see a few edge cases here - the `html`, `head`, and `body` elements are always rendered in the preview, without the user having to enter any code. So the few tests that check those on step-1 of this PR, will never fail. It's not a huge deal, they still have to pass the other tests - but maybe we should stick with regex for those?